tests/e2e-pw/specs/shopper/shopper-myaccount-renew-subscription.spec.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { test, expect, Page } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { config } from '../../config/default';
+import { describeif, getAnonymousShopper } from '../../utils/helpers';
+import * as shopper from '../../utils/shopper';
+import * as navigation from '../../utils/shopper-navigation';
+import { products, shouldRunSubscriptionsTests } from '../../utils/constants';
+import RestAPI from '../../utils/rest-api';
+
+describeif( shouldRunSubscriptionsTests )(
+	'Subscriptions > Renew a subscription in my account',
+	() => {
+		const customerBillingConfig =
+			config.addresses[ 'subscriptions-customer' ].billing;
+
+		let subscriptionId: string;
+		let page: Page;
+
+		test.beforeAll( async ( { browser }, { project } ) => {
+			const restApi = new RestAPI( project.use.baseURL );
```

Comment:
We're using `project.use.baseURL` to get the same base URL Playwright is using while running the tests.